Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/mapping/ExposureWindowRiskCalculationConfigMapper.kt

```diff
@@ -0,0 +1,44 @@
+package de.rki.coronawarnapp.appconfig.mapping
+
+import dagger.Reusable
+import de.rki.coronawarnapp.appconfig.ExposureWindowRiskCalculationConfig
+import de.rki.coronawarnapp.server.protocols.internal.v2.AppConfigAndroid
+import de.rki.coronawarnapp.server.protocols.internal.v2.RiskCalculationParametersOuterClass
+import javax.inject.Inject
+
+@Reusable
+class ExposureWindowRiskCalculationConfigMapper @Inject constructor() :
+    ExposureWindowRiskCalculationConfig.Mapper {
+
+    override fun map(rawConfig: AppConfigAndroid.ApplicationConfigurationAndroid): ExposureWindowRiskCalculationConfig {
+        val riskCalculationParameters = rawConfig.riskCalculationParameters
```

Comment:
We should always have the default app config as the ultimate fallback. If that even does not return risk calculation parameters, it's a race condition. Throw an exception or something else so that we can easily catch this during testing 👍

Reply:
has been adressed, an exception is now thrown if riskCalculationParameters do not exist